plugins/gcp-projects/src/index.ts

```diff
@@ -14,5 +14,9 @@
  * limitations under the License.
  */
 
-export { plugin } from './plugin';
+export {
+  gcpProjectsPlugin,
+  gcpProjectsPlugin as plugin,
```

Comment:
For real?

Reply:
To be removed after a depreciation period?

Reply:
Yep, but, would like to implement plugin discovery before we deprecate it, so that people can just delete their `plugins.ts`